src/FaceDetection/test_face_detection.py

```diff
@@ -0,0 +1,16 @@
+import cv2
```

Comment:
Change file name to main.py because it is the main used for testing this file, test_*.py files are reserved for unit test files